app/code/Magento/InventoryInStorePickup/Test/_files/pickup_location.php

```diff
@@ -0,0 +1,54 @@
+<?php
```

Comment:
There is already fixture to make source -> pickup location: `app/code/Magento/InventoryInStorePickup/Test/_files/source_pickup_location_attributes.php`
Do we really need extra one?

Reply:
Yes